M2/Macaulay2/packages/JSON.m2

```diff
@@ -358,8 +358,5 @@ for tst in sort select(tsts, f -> match("^y_", f)) do (
 close outfile
 ///
 
-TEST(currentPackage#"source directory" | "JSON/test-parse.m2",
-    FileName => true)
-
-TEST(currentPackage#"source directory" | "JSON/test-encode.m2",
-    FileName => true)
+TEST get(currentPackage#"auxiliary files" | "test-parse.m2")
```

Comment:
Reading this line, I would be more confused about where to look for that file than before. You can of course go with whatever solution you prefer in your package.